### PR TITLE
Migrate ejb-mutual-tls example to jakarta EE10

### PR DESCRIPTION
**Pull request by PrarthonaPaul**

### CODE REVIEW

File: ejb-mutual-tls/pom.xml [hunk 1]
```diff
@@ -8,32 +8,50 @@
...
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>jakarta.platform</groupId>
+            <artifactId>jakarta.jakartaee-api</artifactId>
+            <version>10.0.0</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>jakarta.servlet</groupId>
```
Comment (Skyllarr) [inline ejb-mutual-tls/pom.xml:52]: @PrarthonaPaul Just a minor, I think servlet is not used in this example

Comment (PrarthonaPaul) [reply ejb-mutual-tls/pom.xml]: Thanks for catching it! 
I have updated it.

File: ejb-mutual-tls/pom.xml [hunk 2]
```diff
@@ -8,32 +8,45 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
-        <version.org.wildfly.wildfly-ejb-client-bom>26.1.1.Final</version.org.wildfly.wildfly-ejb-client-bom>
-        <version.org.wildfly.security.wildfly-elytron>1.19.0.Final</version.org.wildfly.security.wildfly-elytron>
-        <version.org.jboss.spec.javax.annotation.jboss-annotations-api_1.3._spec>2.0.1.Final</version.org.jboss.spec.javax.annotation.jboss-annotations-api_1.3._spec>
         <version.org.wildfly.plugins.wildfly-maven-plugins>2.0.2.Final</version.org.wildfly.plugins.wildfly-maven-plugins>
         <version.org.apache.maven.plugins.maven-ejb-plugins>3.2.1</version.org.apache.maven.plugins.maven-ejb-plugins>
+        <version.wildfly>31.0.0.Final-SNAPSHOT</version.wildfly>
```
Comment (Skyllarr) [inline ejb-mutual-tls/pom.xml:13]: @PrarthonaPaul Thank you for updating! Just a last total minor, we should not use a SNAPSHOT in the examples. We can use a 30.0.0.Final or 31.0.0.Final when it is released (planned on 24th January)

Comment (PrarthonaPaul) [reply ejb-mutual-tls/pom.xml]: Oh, yes, sorry about that. I changed it for testing purposes. 
should be fixed now!

**Review by Skyllarr — APPROVED** [review]

**Pull request merged**